src/Extensions/JsonExtensions.cs

```diff
@@ -1,95 +1,93 @@
-      }
+            ContractResolver = new CamelCasePropertyNamesWithOverridesContractResolver(),
+            Converters = new List<JsonConverter>
+            {
+                new LineInfoConverter(),
+                new TimeSpanConverter(),
+                new StringEnumConverter()
+                {
+                    CamelCaseText = false
+                },
+                new AdjustToUniversalIsoDateTimeConverter()
+            }
         };
-        public static readonly JsonSerializerSettings MediaTypeFormatterSettings = new JsonSerializerSettings()
+
+        public static readonly JsonSerializerSettings MediaTypeFormatterSettings = new JsonSerializerSettings
         {
             MaxDepth = new int?(512),
             TypeNameHandling = TypeNameHandling.None,
             DateParseHandling = DateParseHandling.None,
             DateTimeZoneHandling = DateTimeZoneHandling.Utc,
             NullValueHandling = NullValueHandling.Ignore,
             MissingMemberHandling = MissingMemberHandling.Error,
-            ContractResolver = (IContractResolver)new CamelCasePropertyNamesWithOverridesContractResolver(),
-            Converters = (IList<JsonConverter>)new List<JsonConverter>()
-      {
-        (JsonConverter) new LineInfoConverter(),
-        (JsonConverter) new TimeSpanConverter(),
-        (JsonConverter) new StringEnumConverter()
-        {
-          CamelCaseText = false
-        },
-        (JsonConverter) new AdjustToUniversalIsoDateTimeConverter()
-      }
+            ContractResolver = new CamelCasePropertyNamesWithOverridesContractResolver(),
+            Converters = new List<JsonConverter>
+            {
+                new LineInfoConverter(),
+                new TimeSpanConverter(),
+                new StringEnumConverter()
+                {
+                    CamelCaseText = false
+                },
+                new AdjustToUniversalIsoDateTimeConverter()
+            }
         };
-        public static readonly JsonSerializer JsonObjectTypeSerializer = JsonSerializer.Create(JsonExtensions.ObjectSerializationSettings);
-        public static readonly JsonSerializer JsonMediaTypeSerializer = JsonSerializer.Create(JsonExtensions.MediaTypeFormatterSettings);
+        public static readonly JsonSerializer JsonObjectTypeSerializer = JsonSerializer.Create(ObjectSerializationSettings);
+        public static readonly JsonSerializer JsonMediaTypeSerializer = JsonSerializer.Create(MediaTypeFormatterSettings);
         public const int JsonSerializationMaxDepth = 512;
-        public static readonly MediaTypeFormatter JsonObjectTypeFormatter;
-        public static readonly MediaTypeFormatter[] JsonObjectTypeFormatters;
-        public static readonly MediaTypeFormatter JsonMediaTypeFormatter;
-        public static readonly MediaTypeFormatter[] JsonMediaTypeFormatters;
+
+        //public static readonly MediaTypeFormatter JsonObjectTypeFormatter;
```

Comment:
If this code is no longer needed, we should remove it.

Reply:
I reactivated the code. As it was public I am not sure it was not used.